codegen/model/values/helm_chart_values.go

```diff
@@ -37,9 +37,10 @@ type UserValues struct {
 
 // document values structure for a container
 type UserContainerValues struct {
-	Image     Image                    `json:"image" desc:"Specify the container image"`
-	Env       []v1.EnvVar              `json:"env" desc:"Specify environment variables for the container. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#envvarsource-v1-core) for specification details." omitChildren:"true"`
-	Resources *v1.ResourceRequirements `json:"resources,omitempty" desc:"Specify container resource requirements. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#resourcerequirements-v1-core) for specification details."`
+	Image           Image                    `json:"image" desc:"Specify the container image"`
+	Env             []v1.EnvVar              `json:"env" desc:"Specify environment variables for the container. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#envvarsource-v1-core) for specification details." omitChildren:"true"`
+	Resources       *v1.ResourceRequirements `json:"resources,omitempty" desc:"Specify container resource requirements. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#resourcerequirements-v1-core) for specification details."`
+	SecurityContext *v1.SecurityContext      `json:"securityContext,omitempty" desc:"Specify container security context. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#securitycontext-v1-core) for specification details."`
```

Comment:
Should we extend the description to give a summary of the default if the field is not set?

Reply:
Oh ok got it. So I think we can actually omit it here, but I will update the helm value references in this [PR](https://github.com/solo-io/gloo-mesh-enterprise/pull/4083) to contain the default values. Those will be user facing on our docs [here](https://docs.solo.io/gloo-mesh-enterprise/latest/reference/helm/gloo_mesh_enterprise/latest/gloo_mesh_enterprise_helm_values_reference/) would that work?